nntrainer/tensor/tensor.cpp

```diff
@@ -501,7 +501,8 @@ Tensor Tensor::getSharedDataTensor(const TensorDim dim_, unsigned int offset,
                                    bool reset_stride) const {
   Tensor ret = *this;
 
-  if (dim_.getDataLen() + offset > dim.getDataLen())
+  if (dim_.getDataLen() * sizeof(float) + offset >
```

Comment:
Good catch.
you can update to using `bytes()` after #1452 :)

Reply:
Done :)